js&css/web-accessible/functions.js

```diff
@@ -150,6 +154,7 @@ ImprovedTube.ytElementsHandler = function (node) {
 	} else if (id === 'movie_player') {
 		if (!this.elements.player) {
 			ImprovedTube.elements.player = node;
+			ImprovedTube.elements.player.stopVideo();
```

Comment:
worth documenting

Reply:
When youtube first open, too many things is loading. From youtube default code to our loop code. Therefore, the pause code sometimes will lag which cause the few milisecond play time. That what I feel

Reply:
That's why I stop the video from loading first until everything is ready

Reply:
oh, my bad! _(it is just like @dodieboy said already & the first commit does nothing else)_
So not to spend ms for everybody else it should be conditional to ImprovedTube.storage.player_autoplay === false?